src/index.ts

```diff
@@ -1,13 +1 @@
-require('dotenv').config();
-import express from 'express';
-
-const app = express();
-const port = process.env.PORT || 3000;
-
-app.get('/', (_req, res) => {
-    res.send('Hello! I am event maker telegram bot');
-});
-
-app.listen(port, () => {
-    console.log(`Bot listening at http://localhost:${port}`);
-});
+import 'Bot';
```

Comment:
>I think it will be a bit confusing just import it? will it somehow impact bundle size? I know that for Nodejs app is not an issue, but just curious.

No, it won't, because we anyway need everything what is in the Bot module. So no matter how it propagates in the project's file hierarchy.

Yes, I like that you've suggested and yes, let's change it next time. But not sure yet if we should go oop approach or functional. Maybe we need to get more familiar with the API of the bot package to see what approach makes more sence.